pyvera/__init__.py

```diff
@@ -284,10 +284,25 @@ def __init__(self, json_obj, vera_controller):
             else:
                 self.name = 'Vera Device ' + str(self.device_id)
 
-    def get_payload_parameter_name(self, name):
+    @staticmethod
```

Comment:
I'm definitely a Python noob, so rather than make any concrete changes in an area that seems vague (the first link says an *advantage* of static methods is that they allow the developer to  override them in subclasses, and the second says their definition is immutable ❓ ), I've backed out this change in 4e9d4e7 and added a comment to disable the lint error it generates instead.